cmake/ctest/drivers/atdm/ats1/local-driver.sh

```diff
@@ -1,22 +1,42 @@
 #!/bin/bash -l
 
-if [ "${SBATCH_TIME_LIMIT_MINUTES}" == "" ] ; then
-  SBATCH_TIME_LIMIT_MINUTES=180  # Default limit is 3 hours
+if [ "${SBATCH_BUILD_TIME_LIMIT_MINUTES}" == "" ] ; then
+  export SBATCH_BUILD_TIME_LIMIT_MINUTES=600 # Default 10 hour time limit
 fi
 
-if [ "${Trilinos_CTEST_DO_ALL_AT_ONCE}" == "" ] ; then
-  export Trilinos_CTEST_DO_ALL_AT_ONCE=TRUE
+if [ "${SBATCH_TEST_TIME_LIMIT_MINUTES}" == "" ] ; then
+  export SBATCH_TEST_TIME_LIMIT_MINUTES=840 # Default 14 hour time limit
```

Comment:
With a 10 hour time limit for the build, we might want to reduce this to 13 hours for a total of 23 hours.  That would avoid any chance of running into the next testing day for this build.  Also, from looking at the experimental builds you ran recently shown [here](https://testing-dev.sandia.gov/cdash/index.php?project=Trilinos&begin=2020-05-07&end=2020-05-10&filtercount=3&showfilters=1&filtercombine=and&field1=buildname&compare1=65&value1=Trilinos-atdm-ats1&field2=buildname&compare2=63&value2=-exp&field3=label&compare3=63&value3=Panzer), that longest wallclock time for testing was 9h54m.  Therefore, a 13 hour time limit should be fine for now.

NOTE: The current cron job we are using is starting these builds exactly 1.5 hours apart so given this setup , we do have almost a full 24 hours to do the configure, build, and run the tests without the jobs running into each other.